.gitignore

```diff
@@ -1,2 +1,8 @@
 .idea/
 .vscode/
+.godot/
+.mono/
+__pycache__/
+.venv
```

Comment:
If possible move these to .gitignores inside engine and/or core directories.

Reply:
Sure, moved most of them. I left some IDE specif ones though.